frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -62,7 +62,8 @@ export interface DashCardProps {
   isFullscreen?: boolean;
   isMobile?: boolean;
   isNightMode?: boolean;
-  isPublic?: boolean;
+  /** If public sharing or static/public embed */
```

Comment:
do we really need to duplicate this comment to every place this flag exists? Does that mean the name itself isn't clear enough? I don't have a strong conviction if we should keep or remove them. 5 of them don't bother me much. But it just sparks this question.

Reply:
I don't love it, but I didn't come up with a better alternative, I know copy pasted things are not perfect but putting the comment only in one place wasn't perfect either.
This makes me wonder if this prop should be a flag on redux 🙃, but that's a bit to invasive of a refactor for now

Reply:
you have a good point about centralizing the prop, but not for this PR. Let's leave it as is for now.